app/utils/logging.js

```diff
@@ -47,7 +47,11 @@ export const downloadLogs = () => {
 // ========== STRINGIFY =========
 
 export const generateLogHeader = () => (
```

Comment:
@SebastienGllmt 
I would propose to add two more items logs:
1. Yoroi user selected display language (can get from profile)
2. Browser current display language (using `navigator.language`)
How do you think ?

Reply:
https://app.clubhouse.io/emurgo/story/934/add-language-and-yoroi-version-to-header-requests exists so let @leivaburto know if you do this

Reply:
@leivaburto 
Regarding: https://app.clubhouse.io/emurgo/story/934/add-language-and-yoroi-version-to-header-requests

I am not updating, in logs.
```
1. Yoroi user selected display language (can get from profile)
2. Browser current display language (using navigator.language)
```
Thanks @SebastienGllmt